github-action-build/setup.sh

```
@@ -28,24 +28,26 @@ if [ ! -d "$TEST_HOME" ]; then
 fi
 
 echo '======================= Extracting Base Products to test home directory======================='
-wget "https://atuwa.private.wso2.com/WSO2-Products/identity-server/6.1.0/wso2is-6.1.0.zip" -O $TEST_HOME/wso2is-6.1.0.zip
+curl -k -L -o "$TEST_HOME/wso2is-6.1.0.zip" \
+            "https://atuwa.private.wso2.com/WSO2-Products/identity-server/6.1.0/wso2is-6.1.0.zip"
 unzip $TEST_HOME/wso2is-6.1.0.zip -d $TEST_HOME
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)